vignettes/cgdsMigration.Rmd

```diff
@@ -17,149 +18,295 @@ output:
 patients can be obtained using the `clinicalData` function.
 
+* Can not query for multiple Studies.
+
+* It is possible to get directly `case_ids`. We can use `samplesInSampleLists` function.
+
+```{r}
+samplesInSampleLists(api = cbio, 
+                     sampleListIds = c("gbm_tcga_pub_expr_classical", "gbm_tcga_pub_expr_mesenchymal") )
+```
+
+**Note**
+
+* We can query multiple `case_list_id` to get `case_ids` (samples identifiers)
+
+* To get more information about Patients, we can query with `getSampleInfo` function.
+
 ```{r}
-clinicalData(cbio, "gbm_tcga_pub")
+getSampleInfo(api = cbio, 
+              studyId = "gbm_tcga_pub",
+              projection = c("SUMMARY", "ID", "DETAILED", "META")) %>%
+    datatable(options = list(pageLength = 5))
 ```
 
-But you can still use a different endpoint to obtain data for a single sample:
+There may be other endpoints to get Info. Please reefer to section [Endpoits and Search options][Endpoits and Search options of `cBioportalData`]
 
 ```{r}
-samplist <- samplesInSampleLists(cbio, clist1)
-onesample <- samplist[["gbm_tcga_pub_all"]][1]
-onesample
+case_ids_1 <- samplesInSampleLists(api = cbio, 
+                                   sampleListIds = "gbm_tcga_pub_expr_classical") %>%
+                unlist() %>%
+                head(1) %>%
+                unname()
+
+
 
 cbio$getAllClinicalDataOfSampleInStudyUsingGET(
-    sampleId = onesample, studyId = "gbm_tcga_pub"
+    sampleId = case_ids_1, 
+    studyId = "gbm_tcga_pub"
 )
+
+
 ```
 
-There may be other endpoints that you could look into: 
+* Marcel, WHEN WE CAN USE THIS ENDPOINTS? It does not returns Clinical data tibble.
+
+# Obtaining Clinical data of Cases/Patients   {.tabset .tabset-fade .tabset-pills}
+
+## `cgdsr`
+
+* The User needs to specify which samples list from one Study.
+
+* `getClinicalData`  uses `case_list_id` as an argument without specifying the `study_id`.
 
 ```{r}
-searchOps(cbio, "clinical")
+getClinicalData.CGDS(x = cgds, 
+                     caseList =  "gbm_tcga_pub_expr_classical") %>%
+    datatable(options = list(pageLength = 5))
+
 ```
 
-# Genetic profiles / molecular profiles
+**Note**
+
+* We queried clinical data for `_expr_classical` cases in `gbm_tcga_pub` study id.
 
-## cgds
+## `cBioPortalData`
 
-```{r,eval=FALSE}
-getGeneticProfiles.CGDS(cgds, cancerStudy = "gbm_tcga_pub")
+* Compared to `cgdsr`, `clinicalData` function needs `studyId` and not `case_list_id` to query clinical data sets.
+
+```{r}
+## get Clinical data
+clinicalData(api = cbio, 
+             studyId = "gbm_tcga_pub") %>%
+   select(sampleId, everything()) %>%
+   datatable(options = list(pageLength = 5))
 ```
 
-## cBioPortalData
+**Note**
+
+* Explicitly we queried a subset (only `_expr_classical`, 54 samples) clinical data with `cgdsr` and all (206 samples) clinical data with `cBioPortalData`.
+
+* **`cBioPortalData` cannot query with `sampleListId` to get a subset of sample list.**
```

Comment:
Yes, it can. Also, you can look up the sample identifiers with `samplesInSampleLists(cbio, "gbm_tcga_pub_expr_classical")`